argparse/argparse_cwl_translation.py

```diff
@@ -7,30 +7,38 @@
 
 from . import cwl_tool as cwlt
 
+PY_TO_CWL_TYPES = {
+    'str': 'string',
+    'bool': 'boolean',
+    'int': 'int',
+    'float': 'float',
+    'list': 'array'
+}
 
 class ArgparseCWLTranslation(object):
 
-    def __init__(self):
+    def __init__(self, generate_outputs=False):
```

Comment:
`generate_outputs` should be documented clearly to avoid confusion: outputs from CWL? outputs from the wrapped program? Definition of `outputs` from CWL (i.e `inp`/`outp`), etc...